.github/workflows/ci.yml

```diff
@@ -204,10 +204,11 @@ jobs:
         uses: ./.github/actions/annotate
 
       - name: Save coverage results
-        uses: actions/upload-artifact@master
```

Comment:
TODO: Revert these changes, use @master instead

Reply:
Changing my mind here, let's stick to a version. We do for actions/download-artifact